hls4ml/writer/vivado_writer.py

```diff
@@ -134,7 +134,7 @@ def _make_array_pragma(variable):
 
     @staticmethod
     def _make_stable_pragma(variable):
-        template = '#pragma HLS STABLE variable={name}'
+        template = '//#pragma HLS STABLE variable={name}'
```

Comment:
I think it would be better to remove the line that calls `_make_stable_pragma` for this, rather than write a commented out line to the HLS.